pkg/asset/backend_dummy.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
```

Comment:
It actually turned out easier to rename the type itself to match the name. Otherwise we have problems testing the object created via `StorageFromConfig` function.